fix(ldap2rdbm): shibboleth-idp getting issue after migration

## ERROR

```
2024-11-19 08:08:56,194 - 127.0.0.1 - DEBUG [net.shibboleth.idp.attribute.resolver.spring.dc.rdbms.impl.RDBMSDataConnectorParser:70] - Data Connector 'siteDataConnector': Parsing v2 configuration [DataConnector: null]
2024-11-19 08:08:56,204 - 127.0.0.1 - ERROR [net.shibboleth.utilities.java.support.service.AbstractReloadableService:233] - Service 'shibboleth.AttributeRegistryService': Reload for shibboleth.AttributeRegistryService failed
net.shibboleth.utilities.java.support.service.ServiceException: org.springframework.beans.factory.BeanDefinitionStoreException: Invalid bean definition with name 'siteDataConnector' defined in null: Could not resolve placeholder 'idp.attribute.resolver.sql.searchFilter' in value "

                %{idp.attribute.resolver.sql.searchFilter}

        "; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'idp.attribute.resolver.sql.searchFilter' in value "

                %{idp.attribute.resolver.sql.searchFilter}

        "
	at net.shibboleth.ext.spring.service.ReloadableSpringService.doReload(ReloadableSpringService.java:377)
Caused by: org.springframework.beans.factory.BeanDefinitionStoreException: Invalid bean definition with name 'siteDataConnector' defined in null: Could not resolve placeholder 'idp.attribute.resolver.sql.searchFilter' in value "

                %{idp.attribute.resolver.sql.searchFilter}

        "; nested exception is java.lang.IllegalArgumentException: Could not resolve placeholder 'idp.attribute.resolver.sql.searchFilter' in value "

                %{idp.attribute.resolver.sql.searchFilter}

        "
	at org.springframework.beans.factory.config.PlaceholderConfigurerSupport.doProcessProperties(PlaceholderConfigurerSupport.java:230)
Caused by: java.lang.IllegalArgumentException: Could not resolve placeholder 'idp.attribute.resolver.sql.searchFilter' in value "

                %{idp.attribute.resolver.sql.searchFilter}

        "
	at org.springframework.util.PropertyPlaceholderHelper.parseStringValue(PropertyPlaceholderHelper.java:180)
2024-11-19 08:08:56,205 - 127.0.0.1 - ERROR [net.shibboleth.idp.profile.impl.ReloadServiceConfiguration:131] - Profile Action ReloadServiceConfiguration: Error reloading service configuration for 'shibboleth.AttributeRegistryService'
2024-11-19 08:08:56,206 - 127.0.0.1 - DEBUG [net.shibboleth.idp.profile.impl.RecordResponseComplete:89] - Profile Action RecordResponseComplete: Record response complete
2024-11-19 08:08:
```

Following doc used to migrate: https://gist.github.com/imShakil/694a0769ebcf679ea63e94e47692103f

--- a/community-edition-setup/tools/ldap_to_rdbm/ldap2rdbm.py
+++ b/community-edition-setup/tools/ldap_to_rdbm/ldap2rdbm.py
@@ -49,6 +49,8 @@ from setup_app.installers.gluu import GluuInstaller
 from setup_app.utils.ldif_utils import myLdifParser
 from setup_app.installers.opendj import OpenDjInstaller
 from setup_app.installers.rdbm import RDBMInstaller
+from setup_app.installers.saml import SamlInstaller
+
 from setup_app.pylib.ldif4.ldif import LDIFWriter
 from ldap3.utils import dn as dnutils
 
@@ -63,6 +65,7 @@ Config.installed_instance = True
 openDjInstaller = OpenDjInstaller()
 rdbmInstaller = RDBMInstaller()
 propertiesUtils = PropertiesUtils()
+samlInstaller = SamlInstaller()
 
 parser = myLdifParser("/opt/opendj/config/schema/77-customAttributes.ldif")
 parser.parse()
@@ -224,6 +227,11 @@ for ldif_fn in (current_ldif_fn, static_ldif_fn):
 
 print("Creating indexes...")
 rdbmInstaller.create_indexes()
+
+if samlInstaller.installed():
+    print("Writing saml persistence configuration")
+    samlInstaller.saml_persist_configurations()
+    samlInstaller.run(['chown', 'jetty:gluu', os.path.join(samlInstaller.idp3ConfFolder, 'datasource.properties')])
 
 print("Writing Gluu config properties")
 rdbmInstaller.rdbmProperties()

